etc/testing/deploy/aws.sh

```diff
@@ -57,7 +63,11 @@ set -x
 case "${OP}" in
   create)
     aws_sh="$(realpath "$(dirname "${0}")/../../deploy/aws.sh")"
-    sudo "${aws_sh}" --region=${REGION} --zone=${ZONE} --state=${STATE_STORE} --no-metrics
+    cmd=("${aws_sh}" --region=${REGION} --zone=${ZONE} --state=${STATE_STORE} --no-metrics)
+    if [[ -n "${CLOUDFRONT}" ]]; then
+      cmd+=(" ${CLOUDFRONT}")
+    fi
+    sudo $cmd
```

Comment:
To expand the array here, I think you'll want to use:
`sudo "${cmd[@]}"`. That syntax expands `cmd` to all of its elements and quotes them, and I believe is generally the way people forward arguments. It's the same as "${@}" if you've ever used that idiom before)